helm/charts/keto/values.yaml

```
@@ -8,7 +8,7 @@ image:
   repository: oryd/keto
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: "v0.6.0-sqlite"
```

I mean without alpha also works but maybe make it most verbose
```suggestion
  tag: "v0.6.0-alpha.1-sqlite"
```